src/main/java/org.wso2.apim.monetization/impl/StripeMonetizationImpl.java

```diff
@@ -606,150 +630,224 @@ public boolean publishMonetizationUsageRecords(MonetizationUsagePublishInfo last
         String apiName = null;
         String apiVersion = null;
         String tenantDomain = null;
+        String applicationName = null;
+        String applicationOwner = null;
         int applicationId;
         String apiProvider = null;
         Long requestCount = 0L;
         Long currentTimestamp;
         int flag = 0;
         int counter = 0;
-        JSONObject jsonObj = null;
         APIAdmin apiAdmin = new APIAdminImpl();
+        SubscriptionItem subscriptionItem = null;
+        String accessToken = null;
+        String queryApiEndpoint = null;
+        String graphQLquery = "query($timeFilter: TimeFilter!, " +
+                "$successAPIUsageByAppFilter: SuccessAPIUsageByAppFilter!) " +
+                "{getSuccessAPIsUsageByApplications(timeFilter: $timeFilter, " +
+                "successAPIUsageByAppFilter: $successAPIUsageByAppFilter) { apiId apiName apiVersion " +
+                "apiCreatorTenantDomain applicationId applicationName applicationOwner count}}";
 
-        DateFormat df = new SimpleDateFormat(StripeMonetizationConstants.TIME_FORMAT);
+        if (config == null) {
+            // Retrieve the access token from api manager configurations.
+            config = ServiceReferenceHolder.getInstance().getAPIManagerConfigurationService().
+                    getAPIManagerConfiguration();
+        }
+        queryApiEndpoint = config.getFirstProperty(StripeMonetizationConstants.ANALYTICS_QUERY_API_ENDPOINT_PROP);
+        if(StringUtils.isEmpty(queryApiEndpoint) ){
```

Comment:
formatting issue?

Reply:
Fixed in 09f0f2a0ec6b148f13a8092bf53055c9df346ef2